CONTRIBUTING.md

```diff
@@ -194,3 +194,36 @@ Each source file needs to have this header:
     // WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
     // See the License for the specific language governing permissions and
     // limitations under the License.
+
+## Quality levels
+
+CMake targets can be developed according to different quality levels. For production usage in a specific environment (e.g. compiler version, compiler flags, etc.), please contact one of the maintainers beforehand. All requirements of a lower quality level are included in higher quality levels e.g. quality level 4 is included in quality level 3.
+
+Also see [ROS quality levels](https://github.com/ros-infrastructure/rep/blob/master/rep-2004.rst).
+
+### Quality level 4
+
+This quality level is the default quality level. It is meant for examples and helper tools.
+
+* Reviewed by two approver
+* License and copyright statement available
+* No version policy required
+* No unit tests required
+
+### Quality level 3
+
+* No compiler warnings
+* Doxygen and documentation available
+* Test specification available
+* Unit tests available
+* Version policy required
+* Level 8 and 9 warnings in Helix QAC addressed
+
+### Quality level 2
+
+* Code coverage statement and branch
+
+### Quality level 1
+
+* Warnings in Helix QAC addressed
```

Comment:
This means no warnings or all warnings are evaluated and maybe some are disabled with the respective code comments?

Reply:
Do we want to specify a way how to handle/describe a disabled warning?

Reply:
It is already described how to disable them with code comments and we also collect the justification

Reply:
> This means no warnings or all warnings are evaluated and maybe some are disabled with the respective code comments?

Yeah, that's meant here. Wanna rephrase it?